hash/bml3_flop.xml

```diff
@@ -17,7 +17,7 @@ Compatibility flags:
 		<sharedfeat name="compatibility" value="5" />
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="346800">
-				<rom name="hiwriter.d88" size="346800" crc="637f3b94" sha1="0bf7fdb528e03aa2409f386e63e257f5aa013c8f" />
+				<rom name="hiwriter.d88" size="346800" crc="53015bf2" sha1="73cbde8f921203335c1afa68ce4accb5d0be4ce5" />
```

Comment:
All these images gets interpreted by [d88_disk.cpp](https://github.com/mamedev/mame/blob/f845a619e5ef7fecac6ea03154609e1f5ff8065e/src/lib/formats/d88_dsk.cpp#L440) as QD, which doesn't look the case by looking at the MP-1802 schematics.

![image](https://github.com/user-attachments/assets/f3381eee-8b84-4e12-87b8-de443813fdd8)